client/src/QnA/AnswerListEntry.jsx

```diff
@@ -20,12 +20,12 @@ function AnswerListEntry({ answer, grabAnswers }) {
 
   const helpfulListener = (e) => {
     e.preventDefault();
-    axios.put('/qa/answers/helpful', { answer_id: answer.answer_id })
+    axios.put('/qa/answers/helpful', { answer_id: answer.id })
       .then((result) => {
         setAnsHelpful(!ansHelpful);
       })
       .then(() => {
-        grabAnswers();
+        grabQuestions();
```

Comment:
In AnswerListEntry, for your helpful (and also report) listener, when you mark an answer as helpful, are you using local storage to ensure that a user can't mark something helpful more than once? It seems like it's updating in state but after refreshing the page I would expect the same user to be able to mark that same answer as helpful again. This would make for a good Jest test.